src/modules/control_allocator/ActuatorEffectiveness/ActuatorEffectivenessTiltrotorVTOL.cpp

```diff
@@ -133,16 +135,12 @@ void ActuatorEffectivenessTiltrotorVTOL::updateSetpoint(const matrix::Vector<flo
 					actuator_sp(i + _first_tilt_idx) += control_collective_tilt;
 				}
 			}
-		}
-
-		// in FW directly use throttle sp
-		if (_flight_phase == FlightPhase::FORWARD_FLIGHT) {
 
-			actuator_controls_s actuator_controls_0;
+			// in FW directly use throttle sp
+			if (_flight_phase == FlightPhase::FORWARD_FLIGHT) {
 
-			if (_actuator_controls_0_sub.copy(&actuator_controls_0)) {
 				for (int i = 0; i < _first_tilt_idx; ++i) {
-					actuator_sp(i) = actuator_controls_0.control[actuator_controls_s::INDEX_THROTTLE];
+					actuator_sp(i) = tiltrotor_extra_controls.collective_thrust_normalized_setpoint;
```

Comment:
This can't come through a regular thrust setpoint now?

Reply:
We would need to hack the API, as a tilted thrust doesn't fit in only one of vehicle_thrust_setpoint.[x]. The clean thing to do would be to give the thrust vector along in vector form, but for that we would need to change quite a bit in tiltrotor.cpp and ActuatorEffectivenessTiltrotor. I would prefer to do this in a follow up as a stand alone task, as it's hard to asses the complexity.